code/controllers/subsystem/air.dm

```diff
@@ -301,6 +305,10 @@ SUBSYSTEM_DEF(air)
 
 	// Clear active turfs - faster than removing every single turf in the world
 	// one-by-one, and Initalize_Atmos only ever adds `src` back in.
+	#ifdef VISUALIZE_ACTIVE_TURFS
+	for(var/turf/active in active_turfs)
+		active.remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, "#00ff00")
```

Comment:
Define for the color and don't type-check in the `for` loop.

Reply:
gottcha

Reply:
yep